[CAPPL-60] Use new EncoderFactory interface in ocr3 capability

bolekk opened the pull request:
Depends on https://github.com/smartcontractkit/chainlink-common/pull/780

bolekk committed:
[CAPPL-60] Use new EncoderFactory interface in ocr3 capability

nolag commented on core/capabilities/encoder_factory.go:
```diff
@@ -0,0 +1,20 @@
+	switch name {
+	case "EVM":
+		return evm.NewEVMEncoder(config)
+	// TODO: add a "no-op" encoder for users who only want to use dynamic ones?
```
Please make a ticket for TODOs and link to it.  Also, custom expresses it better than no-op.  IIUC, it'll be like a call to custom compute.

cedric-cordenier committed:
Common bump

cedric-cordenier committed:
Common bump

0xnogo committed:
Add RMNRemote in the chain reader definition (#14588)

* Update contract_reader.go and go.mod

- Add rmn_remote package to contract_reader.go
- Update chainlink-ccip version in go.mod

* changeset

* Use main branch commit

* introducing logs

* use most recent commit

aalu1418 committed:
solana: add compute unit limit functionality (#14576)

* solana: add compute unit limit functionality

* fix test: solana node CLI

* fix: e2e test artifact upload to container

* retry build with fresh commit

* changeset

* bump solana to merged commit

silaslenihan committed:
go.mod bump
